tests/test_metrics_core.py

```diff
@@ -0,0 +1,83 @@
+import pytest
```

Comment:
Shouldn't this be in test_exposition now?

Reply:
We could move it but it's specific to the classes from `metrics_core.py` so I'm not sure if that would be better

Reply:
It's testing the exposition code though.